File: Sorting/3107.Minimum Operations to Make Median of Array Equal to K/Solution.java
class Solution {
    public long minOperationsToMakeMedianK(int[] nums, int k) {
        int n = nums.length;
        Arrays.sort(nums);
        long ans = 0;
        for (int i = 0; i < n; i++) {
            if (i < n / 2) {
                ans += Math.max(nums[i] - k, 0);
            } else if (i == n / 2) {
                ans += Math.abs(nums[i] - k);
            } else {
                ans += Math.max(k - nums[i], 0);
            }
        }
        return ans;
    }
}

File: Sorting/3107.Minimum Operations to Make Median of Array Equal to K/Solution.py
class Solution:
    def minOperationsToMakeMedianK(self, nums: List[int], k: int) -> int:
        n = len(nums)
        nums.sort()
        ans = 0
        for i in range(n):
            if i < n // 2:
                ans += max(nums[i] - k, 0)
            elif i == n // 2:
                ans += abs(nums[i] - k)
            else:
                ans += max(k - nums[i], 0)
        return ans